example/app/views/sessions/new.html.erb

```diff
@@ -76,7 +76,8 @@
 
   <main class="container" role="main">
     <header>
-      <h1>Shopify App — Installation</h1>
+      <% application_name = ShopifyApp.configuration.application_name %>
+      <h1><%= application_name.presence || 'Shopify App – Installation' %></h1>
```

Comment:
I think we can skip these changes also. I'm going to remove this file since the example app should be using the `shopify_app` engine controllers and views. It shouldn't have its own sessions/new view.